lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+class Human < Player
+  attr_accessor :input
+  
+  def move(input)
+    puts "Please enter a valid position between 1-9:"
+    puts ""
+    puts ""
+    input = gets.to_s.strip
+    input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(input)
  puts "Please enter a valid position between 1-9:"
  gets.strip
end
```